andela/wg-a-team
# Change travis secret key for clean deployment

#### What does this PR do?

It updates the travis secret key that should lead to a clean build

#### How should this be manually tested?

A travis build on travis should successfully deploy the "develop" branch onto heroku
